darwin/future/core/items/set_item_layout.py

```diff
@@ -28,6 +29,12 @@ def set_item_layout(
     Returns:
         JSONType: The response data.
     """
+    if not isinstance(layout, ItemLayout):
+        try:
+            layout = ItemLayout(**layout)
+        except ValueError:
```

Comment:
Do these not throw `ValidationError`, potentially as well?

Reply:
Nice spot, they could indeed

Will except ValidationError as well in these